addons/mail/static/src/new/core/thread_model.js

```diff
@@ -134,7 +135,12 @@ export class Thread {
             }
             if (this.type === "group") {
                 serverData.channel.channelMembers[0][1].forEach((elem) => {
-                    Partner.insert(this._state, elem.persona.partner);
+                    if (elem?.persona?.partner) {
+                        Partner.insert(this._state, elem.persona.partner);
+                    }
+                    if (elem?.persona?.guest) {
```

Comment:
`elem?` is useless, because it's always set (iteratee of `channelMembers[0][1]`, which is always an object)